docs/glossary.md

```diff
@@ -1,11 +1,13 @@
 ## Wikijump Glossary
 
 There are a number of concepts within Wikidot, which can take on a number of different names in casual usage.
-To standardize their usage during development and within the API, this document will describe the standard name, Wikidot's name (if different), and any other aliases.
+To standardize their usage during development and within the API, this document will describe the standard API name, Wikidot's name (if different), and any other aliases.
+
+Please use the Wikijump API term when writing technical documentation, writing code, or utilizing the API. The aim is clarity and disambiguity.
```

Comment:
```suggestion
Please use the Wikijump API term when writing technical documentation or writing code either for Wikijump or in projects that consume the API. The aim is clarity and disambiguity.
```
Suggestion to remove 'utilise' (using the meaning that I assume was intended)

Reply:
Updated here: https://github.com/scpwiki/wikijump/pull/121/files#discussion_r584016005